integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature

```diff
@@ -51,29 +51,62 @@ Feature: SmartMetering Configuration - Firmware
       | TEST1029000000001    | SMR      | 5.2     | V 1.1 | V 1.2 | V 1.3 | V 1.4 | Telit 10.00.154 | BL_012 XMX_N42_GprsV09 | M57 4836 | M00 0000 |
       | TEST1030000000001    | SMR      | 5.5     | V 1.1 | V 1.2 | V 1.3 | V 1.4 | Telit 10.00.154 | BL_012 XMX_N42_GprsV09 | M57 4836 | M00 0000 |
 
-
   @GetFirmwareVersion @GetFirmwareGas
-  Scenario: Get the firmware version from SMR 5.1 gas meter
+  Scenario Outline: Get the firmware version from <protocol> <version> gas meter
     Given a dlms device
-      | DeviceIdentification | TEST1027000000001 |
-      | DeviceType           | SMART_METER_E     |
-      | Protocol             | SMR               |
-      | ProtocolVersion      |               5.1 |
-      | Port                 |              1027 |
+      | DeviceIdentification | <e-meter>     |
+      | DeviceType           | SMART_METER_E |
+      | Protocol             | <protocol>    |
+      | ProtocolVersion      | <version>     |
     And a dlms device
-      | DeviceIdentification        | TEST1027000000002 |
-      | DeviceType                  | SMART_METER_G     |
-      | GatewayDeviceIdentification | TEST1027000000001 |
-      | Channel                     |                 3 |
-      | FirmwareModuleVersionSimple |          19180706 |
+      | DeviceIdentification        | <g-meter>     |
+      | DeviceType                  | SMART_METER_G |
+      | GatewayDeviceIdentification | <e-meter>     |
+      | Channel                     |             3 |
+      | FirmwareModuleVersionSimple |      19180706 |
     When the get firmware version gas request is received
-      | DeviceIdentification | TEST1027000000002 |
+      | DeviceIdentification | <g-meter> |
     Then the firmware version gas result should be returned
-      | DeviceIdentification | TEST1027000000002 |
-      | SimpleVersionInfo    |          00400011 |
+      | DeviceIdentification | <g-meter> |
+      | SimpleVersionInfo    |  00400011 |
```

Comment:
Make SimpleVersionInfo variable from example or better removed expectedVersion from example does not vary

Reply:
Removed from example